src/js/utils/ServiceUtil.js

```diff
@@ -293,6 +293,8 @@ const ServiceUtil = {
                     portMapping.labels = {};
                     portMapping.labels[`VIP_${index}`] = `${general.id}:${lbPort}`;
                   }
+                } else if (lbPort != null && /^\d*$/.test(lbPort) ) {
```

Comment:
Oh man! This `if`-statement with nested `if`s and `else if`s is waaaaaay too complicated. I just wanted to point it out. Not something we need to fix in this PR